rocketpy/Rocket.py

```diff
@@ -689,74 +694,44 @@ def finNumCorrection(n):
                 + (1 / 6) * (Cr + Ct - Cr * Ct / (Cr + Ct))
             )
 
-        # Calculate lift parameters for planar fins
         if not airfoil:
-
-            clalphaSingleFin = Function(
-                lambda mach: 2
-                * np.pi
-                * AR
-                * (Af / Aref)
-                / (2 + np.sqrt(4 + ((beta(mach) * AR) / (np.cos(gamac))) ** 2)),
+            # Defines clalpha2D as 2*pi for planar fins
+            clalpha2D = Function(lambda mach: 2 * np.pi / beta(mach))
+        else:
+            # Defines clalpha2D as the derivative of the
+            # lift coefficient curve for a specific airfoil
+            airfoilCl = Function(
+                airfoil[0], interpolation="linear", extrapolation="natural"
             )
+            if airfoil[1] == "degrees":
+                for i in range(len(airfoilCl)):
+                    airfoilCl[i][0] = np.radians(airfoilCl[i][0])
 
-            clalphaMultipleFins = (
-                liftInterferenceFactor * finNumCorrection(n) * clalphaSingleFin
-            )  # Function of mach number
-
-            # Calculates clalpha * alpha
-            cl = Function(
-                lambda alpha, mach: alpha * clalphaMultipleFins(mach),
-                ["Alpha (rad)", "Mach"],
-                "Cl",
+            # Differentiating at x = 0 and correctign for compressible flow
+            clalpha2D = Function(
+                lambda mach: airfoilCl.differentiate(x=0, dx=0.1) / beta(mach)
```

Comment:
What do you think about using `x = 1e-3` (which means `x = dx`) instead of `x=0`? Since the `.differentiate` method uses [central diferences](https://en.wikipedia.org/wiki/Finite_difference), this makes sure that `airfoilCl` will not be evaluated at `alpha < 0`, as it happens with the current implementation.

I noticed that you added `extrapolation="natural"` when initializing `airfoilCl`, but this may not be useful if the source is a callable (example: `airfoil = (lambda alpha = 2*np.pi/abs(alpha), 'radians')`).

```suggestion
                lambda mach: airfoilCl.differentiate(x=1e-3, dx=1e-3) / beta(mach)
```